package.json

```diff
@@ -9,7 +9,8 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "npm run webdriver-update && protractor",
-    "webdriver-update": "webdriver-manager update --versions.chrome=2.36 --gecko false"
+    "e2e-debug": "npm run webdriver-update && node --inspect $(npm bin)/protractor protractor-debug.conf.js",
+    "webdriver-update": "webdriver-manager update --gecko false"
```

Comment:
do we know if a new version of chromedriver works? versions.chrome 2.36 worked but 2.37 did not.

Reply:
How did it fail before? It seems to be working for me.